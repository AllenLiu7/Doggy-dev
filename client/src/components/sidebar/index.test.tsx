import { mount, shallow } from 'enzyme';

import Component from './index';

describe('sideBar', () => {
  describe('initial state', () => {
    const wrapper = shallow(<Component />);

    test('renders', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });

  describe("after 'show more' button clicked", () => {});
});
