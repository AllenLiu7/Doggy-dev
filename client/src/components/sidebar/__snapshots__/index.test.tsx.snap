// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sideBar initial state renders 1`] = `
<Fragment>
  <styled.div>
    <SideBarLinkList
      show={false}
    />
    <styled.button
      onClick={[Function]}
    >
      Show more
    </styled.button>
    <styled.hr
      width="90%"
    />
    <SideBarFriendList />
  </styled.div>
</Fragment>
`;
